ibis/expr/tests/test_table.py

```diff
@@ -1240,6 +1240,12 @@ def g(x):
     assert_equal(result, expected)
 
 
+def test_add_column_proxies_to_mutate(table):
+    result = table.add_column(ibis.now().cast('date'), name='date')  # fails
```

Comment:
does this actually fail, or what does that comment mean?

Reply:
Just a copy paste from the issue example, fixing it.